server/routes/refer.ts

```diff
@@ -1,29 +1,33 @@
 import type { Router } from 'express'
 
+import Role from '../authentication/role'
+import config from '../config'
 import type { Controllers } from '../controllers'
+// import redirectIfDisabledMiddleware from '../middleware/redirectIfDisabledMiddleware'
 import { referPaths } from '../paths'
-import { RouteUtils } from '../utils'
+import Routes from '../utils/routeBuilder'
 
-export default function routes(controllers: Controllers, router: Router): Router {
-  const { get, post, put } = RouteUtils.actions(router)
+export default function routes(controllers: Controllers): Router {
   const { reasonController, referralsController, peopleController, oasysConfirmationController } = controllers
 
-  get(referPaths.start.pattern, referralsController.start())
-  get(referPaths.new.pattern, referralsController.new())
+  if (config.flags.referEnabled) {
```

Comment:
Alternatively, could pass `redirectIfDisabledMiddleware(config.flags.referEnabled)` to each of the routes as additional middleware. e.g. `.get(referPaths.start.pattern, redirectIfDisabledMiddleware(config.flags.referEnabled), referralsController.start())`